migration-engine/core/Cargo.toml

```diff
@@ -1,30 +1,32 @@
 [package]
-name = "migration-core"
-version = "0.1.0"
 authors = ["Marcus Böhm <boehm@prisma.io>", "Julius de Bruijn <bruijn@prisma.io>"]
 edition = "2018"
+name = "migration-core"
+version = "0.1.0"
 
 [dependencies]
-datamodel = { path = "../../libs/datamodel/core" }
-migration-connector = { path = "../connectors/migration-connector" }
-sql-migration-connector = { path = "../connectors/sql-migration-connector", optional = true }
-user-facing-errors = { path = "../../libs/user-facing-errors" }
+datamodel = {path = "../../libs/datamodel/core"}
```

Comment:
I think now we know who's editor removes the spaces...

Reply:
I will check my cargo extensions.

Reply:
Found the culprit, a TOML extension I used auto formatted incorrectly.